Peacekeeper Borgs

silicons opened the pull request:
Peacekeeper borgs, as detailed in https://tgstation13.org/phpBB/viewtopic.php?f=9&t=6247
It has the following modules:
Flash
"Harm Alarm" that confuses in normal use, 
cookie dispenser (That does not have to be on the borg, as it can work outside of a borg"
a energy bola launcher that has its shots slow anyone hit but wear off after a few seconds
Chem injector to confuse but not outright stun people. 
It can also make fragile energy barriers (Goes down in 1 hit, might be too fragile, but we'll see)
Hug module.

When emagged:
Energy barriers electrocute on touch with a 1 second cooldown
Chem injector with knockout/knockdown drugs
"Harm Alarm" becomes a stunning changeling shriek.. .only it also stuns changelings!
Hug module gains "shock" and "crush" modes. These have a relatively high power cost and a slight cooldown.
Cookie dispenser can be hacked to put a delayed varient of Chloral Hydrate in them.

Thoughts comments concerns? 
:cl:
rscadd: Peacekeeper borgs: Security borgs but without any stuns or restraints! Modules: Harm Alarm, Cookie Dispenser, Energy Bola Launcher, Peace Injector that confuses living things, and a weak holobarrier projector! It also comes with the ability to hug.
rscadd: Cookie Synthesizer: Self recharging RCD that prints out cookies!
tweak: Nanotrasen scientists have added a hugging module to medical and peacekeeper cyborgs to boost emotions during human-cyborg interaction.
/:cl:

silicons committed:
Merge remote-tracking branch 'refs/remotes/tgstation/master'

silicons committed:
Merge remote-tracking branch 'refs/remotes/tgstation/master'

silicons committed:
Merge remote-tracking branch 'refs/remotes/tgstation/master'

silicons committed:
Merge remote-tracking branch 'refs/remotes/tgstation/master'

silicons committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into tgstation/master

silicons committed:
Merge branch 'master' of https://github.com/kevinz000/-tg-station

silicons committed:
Merge pull request #2 from tgstation/master

Update...master?

silicons committed:
Merge pull request #3 from tgstation/master

updates master

silicons committed:
Merge pull request #4 from tgstation/master

updates master

silicons committed:
Peacekeeper Borgs!

silicons committed:
Some fixes

silicons committed:
fixes

silicons committed:
Some more fixes

silicons committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into KeepersOfThePeace

RemieRichards commented:
>I need sprites for the borg itself

no!! false!!

kevinz000 commented:
WHAT THE

kevinz000 commented:
MERGE CONFLICTS? HELP REMIE ;_;

RemieRichards commented:
no! you won't acknowledge the sprites I made so I'm maaaaad.

kevinz000 commented:
oH uhhhh let me uh just go get those... how do I download sprites ;_;

RemieRichards commented:
they're on the forums, in the thread, just click them!

RemieRichards commented on code/game/objects/items/robot/robot_items.dm:
```diff
@@ -146,8 +146,64 @@
+				M.confused += 6
+				M << "<font color='red' size='3'>MY EARS</font>"
+			M << "<font color='red' size='7'>HUMAN HARM</font>"
+		playsound(get_turf(src), 'sound/machines/warning-buzzer.ogg', 70, 3)
```
these aren't even the right sound effects... I'd link them but my college blocks mediafire~ @Iamgoofball and @Cuboos have them too tho.

RemieRichards commented on code/game/objects/items/robot/robot_items.dm:
```diff
@@ -146,8 +146,64 @@
+		cooldown = world.time + 600
+		message_admins("[key_name_admin(user, user.client)](<A HREF='?_src_=holder;adminmoreinfo=\ref[user]'>?</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[user]'>FLW</A>) used an emagged cyborg Harm Alarm in ([user.x],[user.y],[user.z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[user.x];Y=[user.y];Z=[user.z]'>JMP</a>)",0,1)
+		log_game("[user.ckey]([user]) used an emagged Cyborg Harm Alarm in ([user.x],[user.y],[user.z])")
+		return
```
trailing return, remove it please.

RemieRichards commented on code/game/objects/items/robot/robot_items.dm:
```diff
@@ -146,8 +146,64 @@
+		log_game("[user.ckey]([user]) used an emagged Cyborg Harm Alarm in ([user.x],[user.y],[user.z])")
+		return
 
+	return
```
trailing return, remove it please.

RemieRichards commented on code/game/objects/items/weapons/RSF.dm:
```diff
@@ -111,4 +111,74 @@ RSF
+
+/obj/item/weapon/cookiesynth/New()
+	desc = "A self recharging cookie fabricator. It currently holds [matter]/10 cookie-units."
+	return
```
trailing return, remove it please.

RemieRichards commented on code/game/objects/items/weapons/RSF.dm:
```diff
@@ -111,4 +111,74 @@ RSF
+	if(isrobot(user)&&!toxin)
+		toxin = 1
+		user << "Cookie Synthesizer Hacked"
+		return
```
trailing return, remove it please.

RemieRichards commented on code/game/objects/items/weapons/RSF.dm:
```diff
@@ -111,4 +111,74 @@ RSF
+	else if(emagged&&!toxin)
+		toxin = 1
+		user << "Cookie Synthesizer Hacked"
+		return
```
trailing return, remove it please.

RemieRichards commented on code/game/objects/items/weapons/RSF.dm:
```diff
@@ -111,4 +111,74 @@ RSF
+	else
+		toxin = 0
+		user << "Cookie Synthesizer Reset"
+		return
```
trailing return, remove it please.

RemieRichards commented on code/game/objects/items/weapons/RSF.dm:
```diff
@@ -111,4 +111,74 @@ RSF
+		toxin = 0
+		user << "Cookie Synthesizer Reset"
+		return
+	return
```
trailing return, remove it please.

RemieRichards commented on code/modules/reagents/reagent_containers/borghydro.dm:
```diff
@@ -207,3 +207,23 @@ Borg Shaker
+	playsound(loc, 'sound/effects/pop.ogg', 50, 0)
+	var/datum/reagent/R = chemical_reagents_list[reagent_ids[mode]]
+	user << "<span class='notice'>[src] is now dispensing '[R.name]'.</span>"
+	return
```
trailing return, remove it please.

KorPhaeron replied:
This whole thing is copypaste as well

kevinz000 commented:
Can I uhh get a link I'm too noob to find them with search :/

RemieRichards commented:
https://tgstation13.org/phpBB/viewtopic.php?f=9&t=6247&start=400 4dir sprite
https://tgstation13.org/phpBB/viewtopic.php?f=9&t=6247&start=450#p170742 transformation

PKPenguin321 commented:
I thought what's-his-name was already coding this

silicons committed:
trailingreturns

silicons committed:
Sprites!

silicons committed:
Fixeshandsandainotification

silicons committed:
Fixes hands

silicons committed:
Merge branch 'KeepersOfThePeace' of https://github.com/kevinz000/-tg-station into KeepersOfThePeace

kevinz000 commented:
PKPenguin: no idea
Sprites fixed, trailing returns removed :)
Now what's a merge conflict?

Dorsisdwarf commented:
It means that some of the files you edited had a part changed since you downloaded the code to make the PR, but not in a_ part of the file_ you edited.

blockheads commented:
looks really awesome but balancing this will probably be a pretty big pain in the ass with something like this only time will tell.

peoplearestrange commented:
Wasn't Drynwyn going to put the PR up? I'm pretty sure they were on top of things? (https://tgstation13.org/phpBB/viewtopic.php?f=9&t=6247&start=550#p173145)

Bawhoppen commented:
what a horrible pr with confusion
heres a tip: add the code for it and then make a seperate PR to add it to the module list

kevinz000 commented:
IL close this if he puts one up

Robustin commented:
Eh not strong feelings on this one except for "When emagged, checks for changelings".

I would expect you to see the issue with this Mekhi. I'd give it about 1 traitorling round before the captain has a personal ling-detector+stunner.

Cuboos commented:
https://dl.dropboxusercontent.com/u/9654207/HARMLARM.wav
https://dl.dropboxusercontent.com/u/9654207/GermanMemes.wav

Here are the harm sounds i made a while ago, you're gonna have to convert them.

drynwyn commented:
Since it's been mentioned: Yes I was going to put up a PR for this, but Git Stuff took longer than expected and I've been sitting on the completed code without being able to get it comitted (some of what I've done is probably visible under my Git profile). If Kevin got it done that's good for him.

Also, just use the DRAGnet's projectile sprite for energy bolas, it's visually appropriate and is what I used.

optimumtact commented on code/game/objects/items/robot/robot_items.dm:
```diff
@@ -146,8 +146,65 @@
+		for(var/mob/living/M in get_hearers_in_view(9, user))
+			if(iscarbon(M))
+				M.confused += 6
+				M << "<font color='red' size='3'>MY EARS</font>"
```
use visible message instead of your custom speak loop

plus use the standard spans instead of encoding the styles

PKPenguin321 replied:
not to mention these messages suck weenie
make it something like "Your ears ring too loud to hear anything!" or something, these messages are inconsistent with existing ones (and have no punctuation to boot)

optimumtact commented on code/game/objects/items/robot/robot_items.dm:
```diff
@@ -146,8 +146,65 @@
+			M << "<font color='red' size='7'>HUMAN HARM</font>"
+		playsound(get_turf(src), 'sound/machines/warning-buzzer.ogg', 70, 3)
+		cooldown = world.time + 200
+		message_admins("[key_name_admin(user, user.client)](<A HREF='?_src_=holder;adminmoreinfo=\ref[user]'>?</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[user]'>FLW</A>) used a cyborg Harm Alarm in ([user.x],[user.y],[user.z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[user.x];Y=[user.y];Z=[user.z]'>JMP</a>)",0,1)
```
why are you messaging the admins every time someone uses the harm alarm, that will be annoying and pointless.

kevinz000 replied:
Honk blasters have it and spamming it is grief but I'll remove it

optimumtact commented on code/game/objects/items/robot/robot_items.dm:
```diff
@@ -146,8 +146,65 @@
+				M.stuttering += 30
+				M.adjustEarDamage(0, 15)
+				M.Jitter(25)
+				M << "<font color='red' size='3'>AAAAAAAAAAAAGGGGGGGGGGGGGGGHHHHHHHH</font>"
```
VISIBLE MESSAGE

kevinz000 replied:
It's the mobs hearing and not seeing ...?

optimumtact replied:
hearing_message then or wahtever

optimumtact commented on code/game/objects/items/weapons/RSF.dm:
```diff
@@ -111,4 +111,69 @@ RSF
+			return
+	var/turf/T = get_turf(A)
+	playsound(src.loc, 'sound/machines/click.ogg', 10, 1)
+	user << "Fabricating Cookie.."
```
visible message

kevinz000 replied:
Rcds don't have it though

optimumtact replied:
okay

optimumtact commented on code/modules/projectiles/projectile/energy.dm:
```diff
@@ -111,6 +111,29 @@
+	if(iscarbon(target))
+		var/obj/item/weapon/restraints/legcuffs/beartrap/B = new /obj/item/weapon/restraints/legcuffs/beartrap/energy/cyborg(get_turf(target))
+		B.Crossed(target)
+	spawn(40)
```
gross as fuck, use the timer subsystem

Zilenan91 commented:
I still think that this borg is conceptually a terrible idea

kevinz000 commented:
Will do all that when I get home... and try to fix the conflict

silicons committed:
Updates

# Conflicts:
#	code/game/objects/items/weapons/RSF.dm

silicons committed:
Merge remote-tracking branch 'refs/remotes/tgstation/master' into KeepersOfThePeace

silicons committed:
Ohshitifuckedup

kevinz000 commented:
crap, what'd I fuck up this time

silicons committed:
Revert "Ohshitifuckedup"

This reverts commit 89b12fd17c35518938b030919bd64ff63b5d1e31.

silicons committed:
Revert "Revert "Ohshitifuckedup""

This reverts commit c0c4e0fd8e9a6d5e8758d421139abe4803a927ad.

PKPenguin321 commented on code/game/objects/items/robot/robot_items.dm:
```diff
@@ -146,8 +146,67 @@
+				M.Jitter(25)
+				M.visible_message("<font color='red' size='3'>[user] blares out a sonic screech from its speakers!</font>", \
+				"<span class='userdanger'>You hear a sharp screech before your thoughts are interrupted and you collapse, your ears ringing!</span>", \
+				"<span class='danger'>You hear a sonic screech and collapse, your ears riniging!"
```
missing a )

kevinz000 commented:
My DM bugged and is throwing 1.3k errors of things I didn't touch

silicons committed:
fixes2ohboylongday

kilkun commented:
Do ear muffs protect against the harm alarm, emagged or no?

kevinz000 commented:
Not ATM.

silicons committed:
ItWorks1

Wow, it actually works now. Just need to work out some of the
kinks/rebalance as needed.

KorPhaeron commented:
Harm Alarm sounds pretty unfun

PKPenguin321 commented:
^

Same with the rest of the borg, I'm not a fan of this as a concept

Harm alarm sounds particularly awful though

trifornt-tinfornt commented:
Nerf the harm alarm, possibly give it a long cooldown and don't make it too annoying to deal with. I think this borg could be pretty fun on paper but it might not mesh well with gameplay.
I'd want to see it given a shot.

Zilenan91 commented:
Earmuffs nullifying it sounds cool but in practice it'll never be used because there's like 2 ear muffs max on any given station. 2 on Box, 3 on Meta.

xxalpha commented:
Security Cyborg module was voted out of the game by the community. The community did not like the role that the Secborg played.

Why are we adding another Secborg?

No, it does not matter that this module is not as robust as the Secborg, it is intended to play the same role, one of direct harm prevention. The community did not like this and it was removed, so we should not be adding an entirely new cyborg module that is meant to replace something that was voted out.

Bawhoppen commented:
if this gets merged it won't be long til it gets a gun cause "it needs better range combat vroom"

drynwyn commented:
@xxalpha The community DIDN'T dislike the role the secborg played, though. Carbon Security players fill that role on a daily basis, and no-one is yelling that we should remove security (that I know of). The issue was that the Secborg filled that role too effectively, requiring very particular strategies for antagonists to counter.

Bawhoppen commented:
@drynwyn nah you're just wrong

trifornt-tinfornt commented:
I don't think this is supposed to be another Secborg, but rather something that keeps people from being killed. There is literally no way this is going to be able to single handedly defeat antags like the Secborg could. The most it could do would be wading attackers/antags off and saving the person who was under attack.

kevinz000 commented:
Harm alarm has 20 second cooldown normally with 6 seconds of confusion and 1000 power draw per fire.
Emagging it/being a hacked borg makes it 60 seconds of cooldown for a longer confuse time and a bit of deafness, and /ONE SECOND/ of stun. Harm alarm/cookie thingy can be spawned in independently/doesn't need to be in a borg to work.

trifornt-tinfornt commented:
Yeah that doesn't sound like that big of a deal.

Dorsisdwarf commented:
Remember - if the server pop agrees with you, it's a subversion of the community will to go against you. If they disagree with you, they need to fuck off we're not a democracy stop being populist.

kevinz000 commented:
Uh, changelings aren't immune to it. I said it might be a good idea to put it in but apparently not so I won't. Whether it should be protected from by 1. earmuffs or 2. bowman or 3. just use flashbang protection checks which include helmets, that's still to be decided :^)

KorPhaeron commented:
Making security immune to it would be even worse

xxalpha commented:
@dorsisdwarf don't make comments if you haven't been around for the discussion on the forums. I was very opposed to removing the secborg.

NocturnalQuill commented:
Glad to see this finished. If everything is working, we should merge it and test it.

@xxalpha His opinion is just as valid as yours. The main issue with secborg was that people ignored their laws to play mini-sec instead. Yes it was a policy issue, no the admins aren't going to do anything about it. If standard borgs with stun batons aren't a major problem on the server, then this won't be either.

Zilenan91 commented:
If people are making this so that borgs ignoring laws to help sec doesn't happen anymore then why the hell are they making a model that only exists to stop harm. If something only exists to actively stop antagonists from doing things then what the fuck is it going to do but that

peoplearestrange commented:
@Zilenan91 because not only antags cause harm. Hilarity ensues.

xxalpha commented:
Then why remove secborg at all? Silicons are supposed to be effective.

Bawhoppen commented:
>why remove secborg

kilkun commented:
"Possibly make earmuffs negate the effect"

No. Make them negate it. They are in the code for a reason, use them. There is absolutely zero reason for it to negate the Honkerblast 5000 that takes effort to get, but not a cyborg module that is available from roundstart with no effort.

NocturnalQuill commented:
@xxalpha I agree, but the admins have shown that they have no interest in enforcing silicon policy. If they won't fix it with policy, we'll have to fix it with mechanics.

xxalpha commented:
You don't fix it by adding this kind of shit.

Shadowlight213 commented:
Silicon policy is a fucking pain in the ass to enforce. So many goddamn grey areas.
We basically took a look at secborg at the end, and asked "Is this fun overall?"
The answer, was no. Secborg is not fun for the game in general.
I hope there will also be a config option for peacekeeper borgs if they become cancerous?

kevinz000 commented:
Earmuffs get a check when I get to a proper keyboard in which I can type on as I'm on an on screen keyboard. Config option will be in when I do that and ask how on coderbus/someone tells me how here

drynwyn commented:
Config isn't hard, I looked at it earlier. If we're going to have configs for standard and peacekeeper and secborg, though, it makes more sense to create a full config for roundstart borg modules that allows any module to be enabled or disabled, rather than specifically these three.

HankMoodyExperience commented:
CUCK BORG: THE MERGE

silicons committed:
configearmuffs

Peacekeeper borgs now have a config option
"Harm Alarm" now respects and is blocked by earmuffs
Secborg/peaceborg disable config option is now more verbose :^)

NocturnalQuill commented:
@HankMoodyExperience The sprite was originally going to be a literal hugbot

kevinz000 commented:
Configuration and earmuffs up

KorPhaeron commented:
This is test merged. I have no idea if its balanced but it's adorable.

TehZombehz commented:
These are the most adorable things ever, I need them.
There probably needs to be a cooldown on the cookie dispenser. Or don't. Seeing a borg SPEW OUT an excessive amount of cookies made me laugh so hard I almost fell over.

Archie700 commented:
>first testmerge round
>fun borg times
>suddenly BIG BAD HOP comes to subvert them and kill everyone
>watch the hilarity and then suicide when all borgs exploded

HOOOMAN HORM

Shadowlight213 commented:
@kevinz000 I turned myself into a peacekeeper borg in the thunderdome, but was unable to equip any modules.
:|

Shadowlight213 commented:
They also definitely need a cooldown on the shocks the emagged barrier can make.
I used harm intent and shoved a human against it.
Since I kept pushing them, they repeatedly got shocked about 7 or 8 times and were in crit before the knockdown of the electrocute animation even kicked in.
Alternatively could just leave it in, since shocked grill does the same thing :^)

kevinz000 commented:
Fuck it's test merged and I'm at school.

kevinz000 commented:
Screams internally

drynwyn commented:
Played during the testmerge and they are the MOST ADORABLE, merge posthaste and ban people who play them just to be a shitter

kevinz000 commented:
Please do not merge untill I fix the hypospray bugs. Shouldn't take too long.

silicons committed:
Tryingtofixhypoandhugs

peoplearestrange commented:
Hug module is needed. In emag mode should hug slightly too long making you feel really uncomfortable.

kevinz000 commented:
Yus

silicons committed:
hugshockshonkhonk

silicons committed:
cooldowns

kevinz000 commented:
Adding hug module. Removing stun-arm. Stun-arm is overpowered.
Hug module will have stun arm functionality and a bit more. Shock/crush is behind a relatively high energy cost and a slight cooldown to make it not too overpowered.
Shock barrier now has a 1 second cooldown to prevent spamming.
Hug module now has 4 modes. Hug, forceful hug (does nothing except for text message), shock, crush. 
Do not merge as I am still debugging.

silicons committed:
standardborgsgethugstoo

silicons committed:
bugfixes

silicons committed:
Nerfs

silicons committed:
staminaloss capping + actually "balanced" staminatoxin

silicons committed:
woopsies

silicons committed:
woopsies2

NocturnalQuill commented:
Are we going with the diagonal looking sprite sprites? Remie made 90 degree side sprites as well

silicons committed:
stunarmnerf

Stun arms now have a cooldown of 3 seconds.

silicons committed:
NoMessageSpam

Harm Alarm no longer spams multiple lines of text!

silicons committed:
FixesGrammer+otherstuff

RemieRichards commented:
@NocturnalQuill the 90* ones that I <i>really don't want used because I hate them</i>? yeah I wonder why we're not using them :P

PineappleWolf commented:
Issue: Peacekeeper borgs can run out of energy snares and chemicals, they do not recharge.

kevinz000 commented:
Snares is probably fixed in the last push, and chemicals are recharging last I checked? Testing some more later today locally

KorPhaeron commented:
needs fixing first

NocturnalQuill commented:
Balance-wise I think we're in a pretty good place (assuming that emagged peacekeepers got toned the fuck down). Fix the bugs and we should be ready to go.

xxalpha commented:
Balance wise secborgs were also fine.

PKPenguin321 commented on code/modules/reagents/chemistry/reagents/toxin_reagents.dm:
```diff
@@ -700,3 +714,31 @@
+	M.confused += 1
+	M.Dizzy(1)
+	if(prob(20))
+		M << "You feel confused and disorientated."
```
spanclass notice

PKPenguin321 commented:
i fucking hate these abominations

TechnoAlchemisto commented:
same fam

NocturnalQuill commented:
Sorry you two don't like them, but you appear to be significantly in the minority

PKPenguin321 commented:
reminder that there still hasn't been a real poll on these

Zilenan91 commented:
Why does luke cox always talk about percentages despite having literally zero ways of getting these statistics and being wrong over half the time

NocturnalQuill commented:
Everything from the forum thread to the response to this PR to the response to the ingame test indicates that a sizable majority wants this added in some form. I agree what we should run an ingame poll, but if you think that it's going to validate you, I'm afraid I have some bad news.

peoplearestrange commented:
@PKPenguin321 but WHY do you hate them? Other than just "because" or "I miss sec borg", why are these so terrible?

GunHog commented:
I like Peacekeepers, they are a good joke borg, but the loss of the Secborg was a serious blow to both silicons and security. Antags, of course love that, and there is no shortage of people who hate silicons in any form, period.

xxalpha commented:
Why would you subvert the AI now if it doesn't have lasering cyborgs to command?

Axed out an entire way to play traitor.

Shadowlight213 commented:
Secborg was removed because it was utterly unfun for anyone BUT security and silicons.
Also I almost never saw secborg using their laser guns due to the power drain. It was always disabler, cuff, harm baton.
Emagged peacekeeper borg is actually pretty powerful. It just requires a bit more thinking than "SHOOT WITH GUN, BEAT TO DEATH WITH BATON, WIN."

xxalpha commented:
>antags using their TC and winning is bad

huehuehuehu

PKPenguin321 commented:
Emagged secborgs were OP, as a person that used them to escape alone literally two rounds in a row

xxalpha commented:
It's a strategy with a lot of risk and if you can make it pay off, it's not overpowered.

Shadowlight213 commented:
@xxalpha >using tc
You mean just using the boards in the upload, right?

xxalpha commented:
>just

You just press buttons, it's the easiest game.

PKPenguin321 commented:
>a lot of risk

Haha not really

Zilenan91 commented:
Yeah you just click on a secborg three times, with two of them him being unable to do anything, and suddenly you have a guy who can't be stopped by anything but the ENTIRE security force at once and has infinite ammo and stuns for both melee and ranged states

Saying it was anything but overpowered is lying.

PKPenguin321 commented:
yeah seriously the only hard part about subverting borgs is managing to get close enough to emag (not hard if you're roboticist or whatever) and the only hard part of keeping them alive is making sure they're not pants-on-head retarded and at least turn off the apc in the RD's room

NocturnalQuill commented:
Make an in-game poll next time we run a test on these so we settle this once and for all.

Armhulen commented:
So, when are we going to add a warehouse in lavaland with wall-e style secborgs thrown away, angry at HUMENS FOR THROWING THEM OUT

Dorsisdwarf commented:
That sounds amazing.

Armhulen commented:
You fucking bet it does.

optimumtact commented:
Now that we have a cuck shed, we may as well give the cucks a role to play in game.

PKPenguin321 commented:
oranges no

Cuboos commented:
OH BUDDY

On Sun, May 22, 2016 at 8:43 PM, PKPenguin321 <notifications@github.com>
wrote:

> oranges no
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/17355#issuecomment-220881686>
>

Bawhoppen commented:
oranges you have made a big mistake, also there was a big juicy DNM tag

PKPenguin321 commented:
I would like to report a CRITICAL BUG I found with this PR that is currently ruining the servers.

.
.
.
.
.
.
.
.
.

This PR was merged

optimumtact commented:
Don't make me github ban you.

PKPenguin321 commented:
It was a joke :(

TechnoAlchemisto commented:
silicons were a mistake

NocturnalQuill commented:
The salt is delicious

Cuboos commented:
what have we done?

kevinz000 commented:
honk
